sdk/test-utils/recorder/test/sanitizers.spec.ts

```diff
@@ -410,7 +412,7 @@ import { randomUUID } from "@azure/core-util";
       });
     });
 
-    describe("Session-level sanitizer", () => {
+    describe.skip("Session-level sanitizer", () => {
```

Comment:
Why are we skipping this?  Can we add a reason as to why?

Reply:
test proxy has been flaky w.r.t this test, I'll log an issue for this. Unrelated to this PR.

Reply:
#29113